app/models/event.rb

```diff
@@ -3,11 +3,15 @@ class Event < ActiveRecord::Base
   belongs_to :user, :foreign_key => 'creator_id'
   validates :creator_id, :presence => { message: 'Somebody must create this event' }
   validates :name, :presence => { message: 'The event must have a title' }
+  validates_length_of :name, :maximum => 255, :message => 'The event title must be less than 255 characters'
   validates :street, :presence => { message: 'The event must have an address' }
   validates :start, :presence => { message: 'The event must have a start time' }
   validates :start, :date => { after: Time.now, message: 'The event must begin in the future' }, :on => :create
   validates :end, :presence => { message: 'The event must have an end time' }, :date  => { after: :start, message: 'The event must end after the start time' }
   validates :description, :presence => { message: 'The event must have a description' }
+  validates_length_of :description, :maximum => 2000, :message => 'The event description must be less than 2000 characters'
+  validates_length_of :special_instructions, :maximum => 2000, :message => 'The event special instructions must be less than 2000 characters'
```

Comment:
Should specify the length that was passed in (I'm not sure how to off the top of my head).

I'd say get rid of this right now, cause we should have a 1829/2000 type of thing around the description in order to make it easy for the user to know they went past the limit.